tests/test_guest.py

```diff
@@ -326,6 +341,21 @@ def test_seves_with_ovmfx64_and_kernel_should_fail(self):
         self.assertEqual(str(c.exception),
                          "Kernel specified but OVMF doesn't support kernel/initrd/cmdline measurement")
 
+    def test_seves_with_sev_snp_ovmf(self):
+        "Test that sev-es measurement is getting calculated correctly with new sev-snp ovmf"
```

Comment:
I think the comment line can be removed, unless you can add more relevant details.